lib/srv/regular/sshserver.go

```diff
@@ -169,6 +186,10 @@ type Server struct {
 
 	// wtmpPath is the path to the user accounting log.
 	wtmpPath string
+
+	// portForwardingMode is the TCP port forwarding mode allowed
+	// port by this server.
```

Comment:
Nit: Reads a bit weird, is the "port" word needed here?

Reply:
I just randomly like shouting "port". Will delete.